backend/src/api/bitcoin/bitcoin.routes.ts

```diff
@@ -90,7 +90,7 @@ class BitcoinRoutes {
       .get(config.MEMPOOL.API_URL_PREFIX + 'block/:hash/summary', this.getStrippedBlockTransactions);
       ;
 
-      if (config.MEMPOOL.BACKEND !== 'esplora') {
+      if (config.MEMPOOL.BACKEND !== 'none') {
```

Comment:
This would disable these requests when running Bitcoin Core only mode which they are made for
none = bitcoin core only
electrum = romanz/electrs / fulcrum compatible
esplora = blockstream/electrs

Reply:
Makes sense, realized after the fact that it was probs ok to remove...